### PR TITLE
Add noorul find and list

### LINKED ISSUE #50: View room: listRoom
As a staff of the quarantine facility, I can view the list of rooms so that I can view all the room details at a glance.

### LINKED ISSUE #66: UI for findRoom command
The current findRoom command gives the result in the bot's response box. Need to change it to show in the panel list instead for UI.

### PR DESCRIPTION
**Pull request by itssodium**
I have the code that shows the UI for findRoom as well as the code for listRoom and the UI.

### CODE REVIEW

File: src/main/java/seedu/address/logic/Logic.java [hunk 1]
```diff
@@ -51,4 +51,6 @@
      * Set the user prefs' GUI settings.
      */
     void setGuiSettings(GuiSettings guiSettings);
+
+    ObservableList<Room> getRoom();
```
Comment (raymondge) [inline src/main/java/seedu/address/logic/Logic.java]: Nice to have an observablelist for getroom

File: src/main/java/seedu/address/logic/commands/room/ListRoomCommand.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+public class ListRoomCommand extends Command {
+    public static final String COMMAND_WORD = "listroom";
+    public static final String MESSAGE_SUCCESS = "Here are the list of rooms";
+    public static final String NUMBER_OF_ROOMS_UNDEFINED = "Please define number of rooms";
```
Comment (raymondge) [inline src/main/java/seedu/address/logic/commands/room/ListRoomCommand.java]: maybe can change to: "there are no room in the app yet, please define no of rooms"

**Review by raymondge — APPROVED** [review]
Nice work, LGTM!

File: src/main/java/seedu/address/logic/commands/room/ListRoomCommand.java (hunk 2 above)
Comment (itssodium) [reply src/main/java/seedu/address/logic/commands/room/ListRoomCommand.java]: Okay!

**Comment by itssodium** [comment]
Please don't merge even if there are 2 approvals. Thank you!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -175,6 +175,12 @@ Examples:
 * `addtask d/Remind Alice to change bedsheets. r/5` Adds a task with description "Remind Alice to change bedsheets." to Room #5.
 * `addtask d/Running low on masks and needs to be restocked. r/1 dd/12-1-2021` Adds a task with description "Running low on masks and needs to be restocked." and due date "12 Jan 2021 at 12am" to Room #1.
 
+### 3.9 Lists the current rooms: `listRoom`
+
+Lists all the current available rooms.
```
Comment (chiamyunqing) [inline docs/UserGuide.md]: Do you think it is better to write "Lists all the rooms in the application." instead? I got confused with the word 'available' thinking it meant like empty rooms without patients.

File: src/main/java/seedu/address/logic/commands/room/ListRoomCommand.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
+package seedu.address.logic.commands.room;
+
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+public class ListRoomCommand extends Command {
+    public static final String COMMAND_WORD = "listroom";
+    public static final String MESSAGE_SUCCESS = "Here are the list of rooms";
```
Comment (chiamyunqing) [inline src/main/java/seedu/address/logic/commands/room/ListRoomCommand.java]: Do you want to standardise the MESSAGE_SUCCESS with that of list patient command? Like "Listed all rooms."

File: src/main/java/seedu/address/logic/commands/room/ListRoomCommand.java (hunk 2 above)
Comment (chiamyunqing) [reply src/main/java/seedu/address/logic/commands/room/ListRoomCommand.java]: To add on, can even add a "Please initialise the number of rooms using initRooms command".

**Review by chiamyunqing — APPROVED** [review]
LGTM overall! Good job.

File: docs/UserGuide.md (hunk 3 above)
Comment (itssodium) [reply docs/UserGuide.md]: Okay I will change it.

File: src/main/java/seedu/address/logic/commands/room/ListRoomCommand.java (hunk 4 above)
Comment (itssodium) [reply src/main/java/seedu/address/logic/commands/room/ListRoomCommand.java]: Okay.

**Pull request merged**